.github/workflows/pull_requests.yml

```diff
@@ -57,6 +57,6 @@ jobs:
         run: git diff
 
       - name: Upload Codecov Coverage
-        uses: codecov/codecov-action@v1.0.13
+        uses: codecov/codecov-action@v4.0.1
         with:
```

Comment:
Based on the failure in GHA checks, and the requirement for [codecov action](https://github.com/codecov/codecov-action) update, I think the token option is now required. Could I ask Nailgun maintainers @JacobCallahan @jyejare @omkarkhatavkar  to generate the token and upload it as secret, so we can use it here?
```
==> Running command '/home/runner/work/_actions/codecov/codecov-action/v4.0.1/dist/codecov create-commit'
/home/runner/work/_actions/codecov/codecov-action/v4.0.1/dist/codecov create-commit -C 4957762a29a5dc93f0609665298c3928aef2dcf6 -Z
==> Uploader SHASUM verified (05b8[30](https://github.com/SatelliteQE/nailgun/actions/runs/7985527688/job/21804148503?pr=1088#step:9:31)6070b7f242adb37232c5cdb624[31](https://github.com/SatelliteQE/nailgun/actions/runs/7985527688/job/21804148503?pr=1088#step:9:32)ba89958d4c08243e5190a803543b2b  codecov)
info - 2024-02-21 07:35:40,773 -- ci service found: github-actions
Error: Codecov token not found. Please provide Codecov token with -t flag.
Error: Codecov: Failed to properly create commit: The process '/home/runner/work/_actions/codecov/codecov-action/v4.0.1/dist/codecov' failed with exit code 1
```
```suggestion
        uses: codecov/codecov-action@v4.0.1
        with:
            token: ${{ secrets.CODECOV_TOKEN }}
```

Reply:
I think instead the better option is to get rid of codecov altogether.

Reply:
see https://github.com/SatelliteQE/nailgun/pull/1102

Reply:
ack, we can close this PR as we removed the codecov